packages/@okta/vuepress-site/use_cases/inline_hooks/token_hook/token_hook/index.md

```diff
@@ -268,7 +268,13 @@ Returning an error object will cause Okta to return an OAuth 2.0 error to the re
 }
 ```
 
-## Sample Listing of JSON Payload of Response
+## Sample JSON Payloads of Responses
+
+This section provides example JSON payloads for the supported operations.
+
+### Sample Response to Add a Claim
+
+Use the `add` operation to add new claims to a token. If you use the `add` operation and include an existing claim in your response with a different value, that value is replaced. Use the `replace` operation instead. See [Sample Response to Replace an Existing Claim](/use_cases/inline_hooks/token_hook/token_hook/#sample-response-to-replace-an-existing-claim) for more information.
```

Comment:
Attempting to add a system-specific claim or using an invalid operation results in the entire PATCH failing and errors logged in the token hooks events.